tests/console/yast2_samba.pm

```diff
@@ -56,7 +56,7 @@ sub smb_conf_checker {
 
     if ($error ne "") {
         assert_script_run("echo \"$error\" > /tmp/failed_smb_directives.log");
-        return record_soft_failure "bsc#1106876 - Missing smb.conf directives" if (is_sle('>=15') || is_leap('>=15.0'));
+        return record_soft_failure "bsc#1106876 - Missing smb.conf directives";
```

Comment:
Before this was called only for sle 15+, we can ignore leap version, so it makes sense to do to use `is_opensuse` here, or confirm that bug is there for SLE12 as well.

Reply:
I cannot see it for SLE12, so I changed to your suggestion.